packages/hoppscotch-ui/src/components/smart/Placeholder.vue

```diff
@@ -4,24 +4,31 @@
       v-if="src"
       :src="src"
       loading="lazy"
-      class="inline-flex flex-col object-contain object-center mb-4"
+      class="inline-flex flex-col object-contain object-center"
       :class="large ? 'w-32 h-32' : 'w-16 h-16'"
       :alt="alt"
     />
-    <slot name="icon"></slot>
-    <span v-if="heading" class="mb-2 font-semibold text-center">
+    <div v-if="hasIcon">
+      <slot name="icon"></slot>
+    </div>
```

Comment:
Isn't this `div` wrapper unnecessary since no classes are applied, nor does the markup demand it?

Reply:
Nice catch.

Reply:
Resolved in https://github.com/hoppscotch/hoppscotch/pull/3638/commits/c643a68f671cf032842befa426b8077df64f6092

Reply:
The `slot` element, being a placeholder, need not require a condition to render since it's implicitly handled behind the scenes. I've removed the check in https://github.com/hoppscotch/hoppscotch/pull/3638/commits/d518306d11a3da92ab402895623cefd19a528bf1.

Reply:
Cool, thanks